modules/cnf-image-based-upgrade-rollback.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
```

Comment:
It seems like there isn't an instruction mentioning how to cancel a failed upgrade? i.e, moving to idle. Is it intended?

Reply:
No, it's not intended but we have conceptual information about this in the docs. For cancelling an upgrade in progress, we only mention in [the introduction](https://74025--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade#ztp-image-based-upgrade-concept-idle_understanding-image-based-upgrade) that you can move back to the Idle stage to clean up resources until the Upgrade pre-pivot phase.

For a failed upgrade, we have rollback for both gitops and non-gitops way. In this scenario, we emphasize that before moving back to Idle, investigate the possible reasons for failure.

Let me know what you want to see exactly!